Dockerfile

```diff
@@ -12,14 +12,26 @@ ENV WEB_REPO /var/www/html
 # Install required deb packages
 RUN apt-get update && apt-get -y upgrade && \
     rm /etc/apt/preferences.d/no-debian-php && \
-    apt-get install -y php-pear php5-curl php5-mysql php5-json php5-gmp php5-mcrypt php5-ldap php5-gd php-net-socket libgmp-dev libmcrypt-dev libpng12-dev libfreetype6-dev libjpeg-dev libpng-dev libldap2-dev && \
+    apt-get install -y libcurl4-gnutls-dev libgmp-dev libmcrypt-dev libpng12-dev libfreetype6-dev libjpeg-dev libpng-dev libldap2-dev libsnmp-dev && \
     rm -rf /var/lib/apt/lists/*
 
+# Install required packages and files required for snmp
+RUN echo "deb http://ftp.br.debian.org/debian/ wheezy main contrib non-free" > /etc/apt/sources.list && \
+    apt-get update && apt-get install -y snmp-mibs-downloader && \
+    rm -rf /var/lib/apt/lists/* && \
+    curl -s ftp://ftp.cisco.com/pub/mibs/v2/CISCO-SMI.my -o /var/lib/mibs/ietf/CISCO-SMI.txt && \
+    curl -s ftp://ftp.cisco.com/pub/mibs/v2/CISCO-TC.my -o /var/lib/mibs/ietf/CISCO-TC.txt && \
+    curl -s ftp://ftp.cisco.com/pub/mibs/v2/CISCO-VTP-MIB.my -o /var/lib/mibs/ietf/CISCO-VTP-MIB.txt && \
+    curl -s ftp://ftp.cisco.com/pub/mibs/v2/MPLS-VPN-MIB.my -o /var/lib/mibs/ietf/MPLS-VPN-MIB.txt
+
```

Comment:
* Replace debian.ftp.fr by the international URI
* Why do you need the mibs downloader if you don't use it then ? (does phpipam use it indirectly?)
* if you need to install an additional package, please do it in the first RUN (avoid the reupdate and re-remove the apt cache)
* Why do you need these specific MIBs? Is it specific to your use-case? Does it make sense to put them in this image and not to build yours on top of this one (speaking only about these Cisco MIBs).

Reply:
OK, some points are more clear to me now.

Why not replacing
```
RUN apt-get update && apt-get -y upgrade && \
     rm /etc/apt/preferences.d/no-debian-php && \
     apt-get install -y libcurl4-gnutls-dev libgmp-dev libmcrypt-dev libpng12-dev libfreetype6-dev libjpeg-dev libpng-dev libldap2-dev libsnmp-dev && \
     rm -rf /var/lib/apt/lists/*
```
and what follows,
by 
```
RUN apt-get update && apt-get -y upgrade && \
     rm /etc/apt/preferences.d/no-debian-php && \
     sed -i /etc/apt/sources.list -e 's/$/ non-free'/ && \
     apt-get install -y libcurl4-gnutls-dev libgmp-dev libmcrypt-dev libpng12-dev libfreetype6-dev libjpeg-dev libpng-dev libldap2-dev libsnmp-dev snmp-mibs-downloader && \
     rm -rf /var/lib/apt/lists/*
```
So that you install `snmp-mibs-downloader` from jessie by just adding non-free as authorized repos, all in the same line.
Then, you can proceed in another layer to the download of missing MIBs.

```

Reply:
Ok I'll proceed this way.